packages/frontend/editor-ui/src/components/WhatsNewModal.vue

```diff
@@ -0,0 +1,264 @@
+<script setup lang="ts">
+import {
+	DynamicScroller,
+	DynamicScrollerItem,
+	type RecycleScrollerInstance,
+} from 'vue-virtual-scroller';
+import 'vue-virtual-scroller/dist/vue-virtual-scroller.css';
+import dateformat from 'dateformat';
+import { useI18n } from '@n8n/i18n';
+import { VERSIONS_MODAL_KEY, WHATS_NEW_MODAL_KEY } from '@/constants';
+import { N8nCallout, N8nHeading, N8nIcon, N8nLink, N8nMarkdown, N8nText } from '@n8n/design-system';
+import { createEventBus } from '@n8n/utils/event-bus';
+import { useVersionsStore } from '@/stores/versions.store';
+import { computed, ref, watch } from 'vue';
+import { useTelemetry } from '@/composables/useTelemetry';
+import { usePageRedirectionHelper } from '@/composables/usePageRedirectionHelper';
+import { useUIStore } from '@/stores/ui.store';
+
+const props = defineProps<{
+	modalName: string;
+	data: {
+		articleId: number;
+	};
+}>();
+
+const pageRedirectionHelper = usePageRedirectionHelper();
+
+const scroller = ref<RecycleScrollerInstance>();
+
+const i18n = useI18n();
+const modalBus = createEventBus();
+const versionsStore = useVersionsStore();
+const uiStore = useUIStore();
+const telemetry = useTelemetry();
+
+const nextVersions = computed(() => versionsStore.nextVersions);
+
+const openUpdatesPanel = () => {
+	uiStore.openModal(VERSIONS_MODAL_KEY);
+};
+
+const onUpdateClick = async () => {
+	telemetry.track('User clicked on update button', {
+		source: 'whats-new-modal',
+	});
+
+	await pageRedirectionHelper.goToVersions();
+};
+
+watch(
```

Comment:
When are these updated? Do we really need to watch for changes or can we just pass them to the component?

Reply:
With the current implementation they shouldn't really change. I was after a "on component mount, mark the articles visible on this modal read" implementation, how would you rather see that done?